lib/charms/sdcore_nrf/v0/fiveg_nrf.py

```diff
@@ -0,0 +1,308 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Library for the `fiveg_nrf` relation.
+
+This library contains the Requires and Provides classes for handling the `fiveg_nrf`
+interface.
+
+The purpose of this library is to relate a charm claiming to provide
+NRF information and another charm requiring this information.
+
+## Getting Started
+From a charm directory, fetch the library using `charmcraft`:
+
+```shell
+charmcraft fetch-lib charms.sdcore_nrf_operator.v0.fiveg_nrf
+```
+
+Add the following libraries to the charm's `requirements.txt` file:
+- pydantic
+- pytest-interface-tester
```

Comment:
I see you removed this one from PYDEPS, so I assume it is not required.
```suggestion
```

Reply:
Removed!